src/SwipeProfile.css

```diff
@@ -1,5 +1,107 @@
 .SwipeProfile {
-    text-align: center;
-    margin: auto;
-    padding: 20px;
-  }
\ No newline at end of file
+  text-align: center;
+  margin: auto;
+  position: relative;
+}
+ul {
+  /*unordered list for navigation bar*/
+  background-color: #f8dabb;
+  list-style-type: none;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  float: left; /*to get navigation bar items to slide next to each other*/
+}
+
+li a {
+  text-align: center;
+  padding: 14px 16px;
+  display: block;
+  color: black;
+  text-decoration: none;
+}
+
+li a:hover:not(.active) {
+  background-color: rgb(
+    102,
+    102,
+    102
+  ); /*color change when mouse hovers over the list item on navigation bar*/
+}
+
+.active {
+  background-color: #f7dfc7;
+}
+
+.profileIntroSection {
+  /*turquoise section that includes other info in smaller font*/
+  background-color: #4dadb7;
+  align-items: center;
+  margin-right: auto;
+  margin-left: auto;
+  width: 500px;
+  border-radius: 12px;
+}
+
+.letsTalkButton {
+  /*"yes, let's match" button */
+  background-color: #e96a6a;
+  border: 2px solid black;
```

Comment:
add border radius to round the corners and remove the black outline, do this the same for the not interested button please

Reply:
Okkk